2021-10-17 06:01:53,774 ERROR: Exception on /hair/None [GET] [/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/derek/Development/zara_consulting/apps/comececeme/app/blueprints/hair/routes.py", line 74, in get_pattern
    pattern = session.get('pattern').title() or None
AttributeError: 'NoneType' object has no attribute 'title'
2021-10-17 06:02:53,605 INFO: ComeCeCeMe startup [/home/derek/Development/zara_consulting/apps/comececeme/app/__init__.py:117]
2021-10-17 06:03:05,570 ERROR: Exception on /hair/None [GET] [/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/derek/Development/zara_consulting/apps/comececeme/venv/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/derek/Development/zara_consulting/apps/comececeme/app/blueprints/hair/routes.py", line 75, in get_pattern
    pattern = session.get('pattern').title() or None
AttributeError: 'NoneType' object has no attribute 'title'
2021-10-17 09:18:11,959 INFO: ComeCeCeMe startup [/home/derek/Development/zara_consulting/apps/comececeme/app/__init__.py:117]
